scheds/rust/scx_layered/src/bpf/intf.h

```diff
@@ -243,6 +243,7 @@ enum layer_match_kind {
 	MATCH_SCXCMD_JOIN,
 
 	NR_LAYER_MATCH_KINDS,
+	MATCH_IS_GROUP_LEADER,
```

Comment:
`NR_LAYER_MATCH_KINDS`  has to come last because it's used to check bounds on the match enum.